spec/jobs/auction_worker_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'sidekiq/testing'
+
+describe AuctionWorker do
```

Comment:
There's quite a bit of logic that isn't tested here:

https://github.com/andarcabrera/ebay_clone/blob/2f5f7c5b2376e14aaaa8f1d76994cf91fdfd82b8/app/jobs/auction_worker.rb#L7
https://github.com/andarcabrera/ebay_clone/blob/2f5f7c5b2376e14aaaa8f1d76994cf91fdfd82b8/app/jobs/auction_worker.rb#L8
https://github.com/andarcabrera/ebay_clone/blob/2f5f7c5b2376e14aaaa8f1d76994cf91fdfd82b8/app/jobs/auction_worker.rb#L9